src/lib/constants/utils.ts

```diff
@@ -0,0 +1,4 @@
+export function clean(link: string): string {
```

Comment:
I was feeling a little weird about putting utility functions under a `/constants/` directory, so perhaps we can move some code into a new file structure:

```
src/
├─ lib/
│  ├─ qr/
│  │  ├─ utils.ts
│  │  ├─ utils.test.ts
```

In the proposed file structure, you can put the code for the `clean` function, but we can probably be more specific by naming it `normalizeLinkName`. Along with this function, we also need to move the `utils.test.ts` file into the proposed `/src/lib/qr/` directory. This way, you can now delete `/src/lib/constants/utils.ts` and `/src/lib/constants/utils.test.ts`.

```ts
// This function is meant to clean links against data from `$lib/constants/links`.
export function normalizeLinkName(name: string): string {
  return name.replace(/[^a-zA-Z0-9-]/g, '').toLowerCase();
}
```

We can also move the code defining the `genQRCodeSVG` function (from `src/routes/[link=link].svg.ts`) into the proposed `utils.ts` file to organize it better.

Reply:
Okay! In the new commit I restructured the functions and re named the clean link.